pubsublite/driver/driver.go

```diff
@@ -0,0 +1,9 @@
+package driver
+
+import "context"
+
+type Pubsubber interface {
+	Publish(ctx context.Context, topic string, msg []byte) error
+	Subscribe(ctx context.Context, callback func(msg []byte)) error
```

Comment:
This needs to return a handle I think *Subscription, at least for cancelling.

Reply:
Cancellation is handled by the context. Isn't that enough?

Reply:
So each subscription would need a separate context to only cancel that one?

Reply:
That would be one way, but also multiple subscriptions could share a context and all of them could be stopped by cancelling that context.

Reply:
Also this is for consistency with the rest of the Go CDK.

Reply:
ok fair enough. CDK all in on Contexts it seems. ;)